src/solver/impls/slepc/slepc.cxx

```diff
@@ -606,10 +606,10 @@ void SlepcSolver::monitor(PetscInt its, PetscInt nconv, PetscScalar eigr[],
       output<<formatEig(reEigBout,imEigBout)<<endl;
       if(eigenValOnly){
         simtime=reEigBout;
-        dump.write();
+        bout::globals::dump.write();
```

Comment:
Not for this PR, but it might be nice for `SlepcSolver` to have it's own output file (which is `bout::globals::dump` by default)?